helpers/SurveyLauncher.js

```diff
@@ -7,14 +7,16 @@ module.exports = class SurveyLauncher{
 
 	async launch(target){
 		let introMessage = [
-			`:exclamation: Hello ${target} :exclamation: Welcome to Cathartes.  We are a Destiny 2 clan that wants to make playing the game more fun.  Here are some quick bullet points on what we are about:`,
+			`:exclamation: Hello ${target} :exclamation: Welcome to Cathartes.
+			 We are a Destiny 2 clan that wants to make playing the game more fun.
+			 Here are some quick bullet points on what we are about:`,
```

Comment:
I didn't get a chance to test the display of this yet, but do ES6 templates make newlines anytime the template has newlines, or do you have to specify `\n`? I was only breaking this line up to be more readable and don't want it breaking to multiple lines in Discord.

Reply:
You could also choose to use a string array. Discord.js will concatenate string arrays with newlines when sending them.